ocean_lib/web3_internal/contract_utils.py

```diff
@@ -53,4 +53,22 @@ def get_contracts_addresses(
     if network_addresses is None and network in network_alias:
         network_addresses = addresses.get(network_alias[network], None)
 
+    return _checksum_contract_addresses(network_addresses=network_addresses)
+
+
+@enforce_types
+def _checksum_contract_addresses(
```

Comment:
I would like to see thorough tests for this function.

Reply:
I see you added a test, but it hardly covers all the cases. It is a very simplistic test case and sort of a "happy flow". I want to see all cases covered, with full coverage on this function, specifically. Please test thoroughly that the entire function works as expected.